lib/dynflow/actors/execution_plan_cleaner.rb

```diff
@@ -43,7 +43,7 @@ def start
         def clean!
           plans = @world.persistence.find_old_execution_plans(Time.now.utc - @max_age)
           report(plans)
-          @world.persistence.delete_execution_plans(uuid: plans.map(&:id))
+          @world.persistence.delete_execution_plans(uuid: plans.map(&:id), **{})
```

Comment:
Should we have `**{}` here? I mean, `delete_execution_plans` looks to have `def delete_execution_plans(filters, batch_size = 1000, backup_dir = nil)`. It won't hurt, but it just seems unnecessary. On the other hand, I've noticed this: https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/debug/telemetry/persistence.rb#L34-L38. I guess it makes more sense to add **kwargs support there just in case.